Source/Details/ASPINRemoteImageDownloader.h

```diff
@@ -40,6 +40,16 @@ NS_ASSUME_NONNULL_BEGIN
  */
 + (void)setSharedImageManagerWithConfiguration:(nullable NSURLSessionConfiguration *)configuration;
 
+/**
+ * Sets a custom perconconfigured PINRemoteImageManager that will be used by @c ASNetworkImageNodes and @c ASMultiplexImageNodes
+ * while loading images off the network. This must be specified early in the application lifecycle before
+ * `sharedDownloader` is accessed. If nil is passed in as the PINRemoteImageManager, it will call
+ * setSharedImageManagerWithConfiguration with nil configuration.
```

Comment:
> it will call setSharedImageManagerWithConfiguration with nil configuration.

Does it? How about:

> it will create a default image manager with a `nil` session configuration.

Reply:
You are rrright.